apps/api/src/domain/models/audio-visual/shared/commands/transcripts/add-participant-to-transcript/participant-added-to-transcript.event-handler.ts

```diff
@@ -0,0 +1,34 @@
+import { Inject } from '@nestjs/common';
+import { ICoscradEventHandler } from '../../../../../../../domain/common';
+import { QUERY_REPOSITORY_PROVIDER_TOKEN } from '../../../../../../../domain/models/shared/common-commands/publish-resource/resource-published.event-handler';
+import { AggregateId } from '../../../../../../../domain/types/AggregateId';
+import { AudiovisualResourceType } from '../../../../audio-item/entities/audio-item-composite-identifier';
+import { TranscriptParticipant } from '../../../entities/transcript-participant';
+import { ParticipantAddedToTranscript } from './participant-added-to-transcript.event';
+
+interface IRepository {
```

Comment:
I wonder if we can find a way to compose these to build the AudioVisual repo interfaces.